Emby.Server.Implementations/LiveTv/EmbyTV/RecordingHelper.cs

```diff
@@ -48,12 +49,18 @@ public static string GetRecordingName(TimerInfo info)
 
                 if (!string.IsNullOrWhiteSpace(info.EpisodeTitle))
                 {
+                    var tmpName = name;
                     if (addHyphen)
                     {
-                        name += " -";
+                        tmpName += " -";
+                    }
+                    // Calculate the length of the resulting filename
+                    var recordingNameLength = Encoding.UTF8.GetByteCount(tmpName + info.EpisodeTitle);
```

Comment:
It's probably better to `GetByteCount` for each string individually so you don't need to allocate the full string

Reply:
I agree, but taking into the considerations that this will add more if clauses, I would argue this will make the code more unreadable and less understandable.

Reply:
How will it add more if clauses? Just change the line to

```C#
                     var recordingNameLength = Encoding.UTF8.GetByteCount(tmpName) + Encoding.UTF8.GetByteCount(info.EpisodeTitle);

Reply:
I misundestood your first suggestion, sorry! Made the change like you suggested!